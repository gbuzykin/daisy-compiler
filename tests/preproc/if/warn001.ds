#if false
#elif true
#include "warn001.dsh"
