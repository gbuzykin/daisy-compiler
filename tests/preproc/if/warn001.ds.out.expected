In file included from ./preproc/if/warn001.ds:3
./preproc/if/warn001.dsh:1:2: warning: `#if` without `#endif`
 1 | #if false
   |  ^~
./preproc/if/warn001.ds:1:2: warning: `#if` without `#endif`
 1 | #if false
   |  ^~
./preproc/if/warn001.ds: info: warnings 2, errors 0
